### PR TITLE
Reviewed azs-ref-overview

### PR DESCRIPTION
**Pull request by wturner123**
Reviewed article as per JIRA ticket AS-1398. Have made various changes throughout.

### CODE REVIEW

File: articles/azure/azs-ref-overview.md [hunk 1]
```diff
@@ -158,23 +170,23 @@ You can access UKCloud for Microsoft Azure from the following location:
 
 For more information about Azure Stack Hub, see the following Microsoft resources:
 
-- [*Get Started with Azure*](https://azure.microsoft.com/en-gb/get-started/)
-
-- [Configure the Azure Stack Hub user's PowerShell environment](azs-how-configure-powershell-users.md)
+- [Get Started with Azure](https://azure.microsoft.com/en-gb/get-started/)
```
Comment (suehighmoor) [inline articles/azure/azs-ref-overview.md:173]: Above it says information about Azure Stack Hub, but the link is to Azure doc...

Comment (wturner123) [reply articles/azure/azs-ref-overview.md]: Updated link in commit 7826a84.  Thank you for the review :)

File: articles/azure/azs-ref-overview.md [hunk 2]
```diff
@@ -120,33 +132,33 @@ When designing your UKCloud for Microsoft Azure applications, consider the follo
       Select-Object -Property Type, Version | Format-Table -AutoSize
     ```
 
-- Use load balancers to make the best use of IP address quota
+- Use load balancers to make the best use of IP address quota.
 
-- Azure Stack Hub API may be a couple of releases behind Azure
+- Azure Stack Hub API may be a couple of releases behind Azure API.
```
[the remark this answers is not in the document]
Comment (dbrennand) [reply articles/azure/azs-ref-overview.md:137]: Discussed with @cblackuk. Should be "Azure Stack Hub API may be a couple of releases behind the Azure API."

File: articles/azure/azs-ref-overview.md [hunk 3]
```diff
@@ -120,33 +132,33 @@ When designing your UKCloud for Microsoft Azure applications, consider the follo
...
+- Put large volume data on UKCloud's [*Cloud Storage*](cs-sd.md).
 
-- Clean up backups to conserve storage
+- Clean up backups to conserve storage.
 
-- Consider use of temporary storage
+- Consider use of temporary storage.
 
-- There is a difference in maximum blob storage between Azure and Azure Stack Hub
+- For some blob types, there is a difference in maximum storage between Azure and Azure Stack Hub.
```
[the remark this answers is not in the document]
Comment (wturner123) [reply articles/azure/azs-ref-overview.md]: Changed in commit 0148319

File: articles/azure/azs-ref-overview.md [hunk 4]
```diff
@@ -33,35 +33,35 @@ Virtual machines (VMs) provide the basic building block providing compute in Azu
...
-| Azure Queue Storage         | Message queue as a service to provide asynchronous scale out micro-services, among other things |                                                                                          |
-| Azure Table Storage         | NoSQL database service; good for highly scalable, eventually consistent database support        | Max sizes differ                                                                         |
+| Azure Blob Storage (block)  | Object storage; good for storing documents, videos, images and so on. | Maximum block blob size is 4.75 TB (100 MB x 50,000 blocks) for the 1802 update or newer version. Previous versions are limited to 50,000 x 4 MB (approximately 195 GB). Blob name size differs slightly at 880 characters (1,760 bytes), compared to 1,024 characters (2,048 bytes) for public Azure. |
+| Azure Blob Storage (page) | Used to support virtual machine disks; good for randomly accessed large files. | Maximum page blob size differs greatly at 1TB for Azure Stack, compared to 8TB for public Azure. |
```
Comment (niall-lefley) [inline articles/azure/azs-ref-overview.md:61]:
```suggestion
| Azure Blob Storage (page) | Used to support virtual machine disks; good for randomly accessed large files. | Maximum page blob size differs greatly at 1TB for Azure Stack Hub, compared to 8TB for public Azure. |
```

**Review by cblackuk — APPROVED** [review]
LGTM

**Pull request merged**